framework/src/org/checkerframework/common/signatureinference/SignatureInferenceScenes.java

```diff
@@ -0,0 +1,726 @@
+package org.checkerframework.common.signatureinference;
```

Comment:
An additional subtlety (worth documenting) is that in the above example, signature inference added a @Nullable annotation to unchecked code (i.e. code outside the scope of an @AnnotatedFor). This could be OK, as the user may have run the signature checker with the intent of helping them complete their annotation of the class, but it may be confusing to a user who thought that code outside of an @AnnotatedFor would not be touched by signature inference because by omitting @AnnotatedFor, the user is saying "I have not finished annotating this code yet".